grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java

```diff
@@ -70,8 +71,7 @@
   private int maxSize = DEFAULT_MAX_CHANNEL;
   private int maxConcurrentStreamsLowWatermark = DEFAULT_MAX_STREAM;
 
-  @VisibleForTesting
-  final Map<String, AffinityConfig> methodToAffinity = new HashMap<String, AffinityConfig>();
+  @VisibleForTesting final Map<String, AffinityConfig> methodToAffinity = new HashMap<>();
```

Comment:
readability nit: this line has "@VisibilityForTesting" on the same line but the next line has it on its own line. I think you should stick one of the 2 styles and be consistent..

Reply:
Yeah, I know, but this is how `gradle googleJavaFormat` works.